src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.serviceimpl.FruitTransactionValidatorImpl;
+import java.util.Optional;
+
+public class ReturnOperationHandler implements OperationHandler {
+    private FruitTransactionValidatorImpl validator;
```

Comment:
The validator field is not initialized and will cause a NullPointerException when its methods are called in the apply method. Consider initializing it in the constructor or passing an already initialized validator instance to the apply method.